app/screens/AddEmission/AddEmissionScreen.navigationOptions.tsx

```diff
@@ -12,7 +11,7 @@ const navigationOptions = () => ({
   },
   headerBackTitleVisible: false,
   headerTintColor: Colors.grey100,
-  headerRight: () => <InfoButton />,
```

Comment:
`InfoButton` was better naming than `EmissionInfoButton` (EmissionInfoButton opening methodology screen seems strange to me). Let's revert